From 02f9e84709b4e4756f3e512dede3430f3538fbf5 Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Sat, 22 Aug 2015 23:13:33 +0200
Subject: gdbus: Add a missing include

This fixes the build on non-unix platforms, such as win32 where
gunixfdlist.h is not included.

diff --git a/gio/gdbusmethodinvocation.c b/gio/gdbusmethodinvocation.c
index c5224be..91a7e39 100644
--- a/gio/gdbusmethodinvocation.c
+++ b/gio/gdbusmethodinvocation.c
@@ -29,6 +29,7 @@
 #include "gdbusintrospection.h"
 #include "gdbuserror.h"
 #include "gdbusprivate.h"
+#include "gioerror.h"
 
 #ifdef G_OS_UNIX
 #include "gunixfdlist.h"
-- 
cgit v0.10.2

